adafruit_displayio_sh1107.py

```diff
@@ -141,7 +146,7 @@ class SH1107(displayio.Display):
 
     def __init__(
         self,
-        bus: displayio.I2CDisplay,
+        bus: Union[displayio.I2CDisplay, displayio.Fourwire],
```

Comment:
Looks like a typo, this should be `FourWire`!

Reply:
yes, I agree - my mistake